sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -147,18 +151,126 @@ _rom_ext_start_boot:
    * If an exception fires, the handler is conventionaly only allowed to clobber
    * memory at addresses below `sp`.
    */
-  la   sp, (_stack_end - 16)
+  
+  lui sp, %hi(_stack_end - 16)
+  addi sp, sp, %lo(_stack_end - 16)
+  
+  /**
+   * Initialize the Ibex simple address translation to run position dependent
+   * code in the virtual flash space.
+   */
+  /**
+   * Compute the current slot address by: 
+   * `addr = pc - _addr_translation_crt_offset`
+   */
+  lui t0, %hi(_addr_translation_crt_offset)
```

Comment:
Can you add some explanation for the lack of pseudo instructions? I don't think it is going to be very clear to people reading the code later why we aren't using them otherwise.

Reply:
Done